src/Microsoft.Azure.WebJobs.Host/Bindings/SystemBindingData.cs

```diff
@@ -36,14 +36,14 @@ internal class SystemBindingData
         public string MethodName { get; set; }
 
         /// <summary>
-        /// Get the current UTC date. 
+        /// Get the current UTC date. This returns the current value each time it is called.
         /// </summary>
-        public DateTime UtcNow => DateTime.UtcNow;
+        public DateTime DateTime => DateTime.UtcNow;
 
         /// <summary>
         /// Return a new random guid. This create a new guid each time it's called.
         /// </summary>
-        public Guid RandGuid => Guid.NewGuid();  
```

Comment:
>RandGuid [](start = 20, length = 8)

Ok. Although, I kind of prefer having 'RandGuid' since it's clearer about the source, and less ambiguous about which guid we're referring to.